pkg/domain/domain.go

```diff
@@ -2238,7 +2238,7 @@ func (do *Domain) UpdateTableStatsLoop(ctx, initStatsCtx sessionctx.Context) err
 	}
 	do.statsHandle.Store(statsHandle)
 	do.ddl.RegisterStatsHandle(statsHandle)
-	// Negative stats lease indicates that it is in test, it does not need update.
+	// Negative stats lease indicates that it is in test or in br binary mode, it does not need update.
```

Comment:
From where can we see stats is set negative for br binary mode?

Reply:
https://github.com/pingcap/tidb/blob/a51ea16fb7e3b9d434f01f44bc33fb15315f64a6/br/cmd/br/backup.go#L50